### DISCUSSION TITLE
quarkus-quartz, how to create 1 clustered job (jdbc-cmt store) and 1 non clustered job (ram store) in a single app.

### DISCUSSION THREAD

**Discussion by mapuci** [quarkusio/quarkus#41456] [Q&A] [answered]
example:
job A needs to be scheduled every 10seconds on ALL pods. (this is RAM store type, that is how quarkus-scheduler works)
job B needs to be scheduled every 20seconds on a single pod. (jdbc-cmt store type, clustered mode, quarkus-quartz only)

I can't figure it out. Any ideas welcome!

[a comment by an automated account is not shown]

**Comment by mkouba** [comment, marked as answer]
This is not supported ATM. You can only choose one of those in a single Quarkus app.
